WeatherBotService/WeatherBotStation/Data/WeatherDataObservable.cs

```diff
@@ -0,0 +1,32 @@
+using WeatherBotStation.WeatherBots;
+using WeatherBotStation.WeatherBots.BotManager;
+
+namespace WeatherBotStation.Data;
+
+public class WeatherDataObservable(IWeatherBotManager manager) : IWeatherDataObservable
+{
+    private readonly IList<IWeatherBot> _bots = manager.GetBots();
```

Comment:
it seems the **GetBots()** method returns only activated bots, what about giving it a more descriptive name? 
 e.g **GetActivatedBots()**